### DISCUSSION TITLE
Mobile version

### DISCUSSION THREAD

**Discussion by danicaso-vizzuality** [Vizzuality/mangrove-atlas#312] [Show and tell]
Let me share here the mobile version we designed to fit the recent updates in the desktop view: https://www.figma.com/proto/qDLzcnVw8uKrm6IFKnS7tM/Mangrove-Watch-mobile-v02.1?page-id=23%3A0&node-id=91%3A6314&viewport=209%2C108%2C0.25&scaling=min-zoom&starting-point-node-id=91%3A6314

With this, we also improved some navigation issues we found in the previous version.

**Comment by lyubovpencheva** [comment]
This is an old one.
